### DISCUSSION TITLE
Commits show up under master branch commits but NOT in Contributors

### DISCUSSION THREAD

**Discussion by DonThePhan** [community/community#29311] [New to GitHub]
Why are my commits showing up under the master branch but not in the contributors section?

It's only showing like 5 commits (out of 137, plus or minus (I'm still trying things so it'll start climbing), that I counted manually in the master branch). Only my commits made after we finished the bulk of our team project were captured under Contributors :(

https://github.com/kivakiva/story-time/graphs/contributors
![Screen Shot 2022-08-12 at 3 29 05 PM](https://user-images.githubusercontent.com/74743983/184429965-78324027-9390-48b3-a61b-ced03503f5e8.png)

Here's a screenshot of just the first page. My username (DonThePhan) is everywhere if you start flipping through the pages. One thing that's apparent is that for the commits that aren't captured, my name doesn't hyperlink to an account, but it's still shows my name. Is this the same issue as not syncing your local account user email w/ your GitHub email? I don't recall having to re establish my email on my local nor gh and it looks like it's being captured currently.

https://github.com/kivakiva/story-time/commits/master
![Commits](https://user-images.githubusercontent.com/74743983/184430109-463911b6-bda6-43eb-9df1-c905d51d5a7d.png)

Can anyone shine some light on what's going on here?

I really just want the Contributors section to reflect the work I've put into this project as I'm using it as an example to get my foot in the industry.

Thanks!

**Comment by airtower-luna** [comment]
The commits that aren't recognized as yours seem to have a broken author email address. For example [the top one in your screenshot](https://github.com/kivakiva/story-time/commit/3dcc5b0835bb88746aa1ba4c313d88fda46523dc.patch):

```
From 3dcc5b0835bb88746aa1ba4c313d88fda46523dc Mon Sep 17 00:00:00 2001
From: DonThePhan <donthephan>
Date: Tue, 1 Mar 2022 18:06:57 -0500
Subject: [PATCH] adjust css to render with 390px for all screens
```

The part between the angle brackets should be your email address, but it isn't. GitHub uses the author email address recorded in the commit to detect that a commit is yours. It must match one of the addresses on your account to be recognized.

To fix this for future commits, set the right email address in your Git configuration as described in [Setting your commit email address](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-email-preferences/setting-your-commit-email-address).

Considering your past commits are already merged in a repository shared with others it's probably not possible to fix them any more, because doing that would require [rewriting repository history](https://git-scm.com/book/en/v2/Git-Tools-Rewriting-History).
